src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
```

Comment:
You are using raw types here. According to the checklist, you should always parametrize interfaces and classes if it is possible. Please, parametrize `MachineProducer` with `Bulldozer`.